css: Remove `.thinner` class used in Settings > Preferences.

saubhagya-patel opened the pull request:
This PR removes the `thinner` CSS class from the "Play animated images" setting. It applies the class to the "Show unread counts for" setting, since the next setting, "Show counts for starred messages" is related to it.

Hence, the bottom margin for the "Play animated images" setting is changed to `20px` (initially it was `10px`). 
For the "Show unread counts for" setting, the bottom margin is set to `10px` (initially it was `20px`), since the next setting, "Show counts for starred messages", is related to it.

<!-- Describe your pull request here.-->

Fixes: [#34226 (comment)](https://github.com/zulip/zulip/pull/34226#discussion_r2026449812).

<!-- If the PR makes UI changes, always include one or more still screenshots to demonstrate your changes. If it seems helpful, add a screen capture of the new functionality as well.

Tooling tips: https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html
-->

**Screenshots and screen captures:**

- In Personal settings > Preferences:

| Description | Before | After |
|-------------|--------|-------|
| "Home view" setting | ![image](https://github.com/user-attachments/assets/f32ae22e-4eab-4901-80e3-276131411606) | ![image](https://github.com/user-attachments/assets/451f8e87-f8ff-43e0-87df-ec191e991057) |
| "Play animated images" setting | ![image](https://github.com/user-attachments/assets/a3ad53e4-7150-4aaf-9d7c-a84d7ef8e36e) | ![image](https://github.com/user-attachments/assets/db78b13d-e32a-4642-b371-61289fca58a0) |

- In Organization settings > Default user settings:

| Description | Before | After |
|-------------|--------|-------|
| "Home view" setting | ![image](https://github.com/user-attachments/assets/a88a1d99-b3ba-4ac0-ad24-fddd14900735) | ![image](https://github.com/user-attachments/assets/6be21010-bec8-479f-96c5-1b280cf4b584) |
| "Play animated images" setting | ![image](https://github.com/user-attachments/assets/5594441e-eb94-49dd-8ac8-915f02150330) | ![image](https://github.com/user-attachments/assets/86724f98-2973-4f47-b067-98e41e79421e) |


---

<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [x] Explains differences from previous plans (e.g., issue description).
- [ ] Highlights technical choices and bugs encountered.
- [ ] Calls out remaining decisions and concerns.
- [ ] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html)).

- [x] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Responsiveness and internationalization.
- [x] Strings and tooltips.
- [x] End-to-end functionality of buttons, interactions and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.
</details>

alya commented:
I think we just want even spacing between all these fields.

saubhagya-patel commented:
`Alya Abbott` [said](https://github.com/zulip/zulip/pull/34389#issuecomment-2802392597):
> I think we just want even spacing between all these fields.

In Settings > Preferences > Navigation ([`zulip/web/templates/settings/preferences_navigation.hbs`](https://github.com/zulip/zulip/blob/main/web/templates/settings/preferences_navigation.hbs#L7)), we also have an instance of the `thinner` class. It is applied to the "Home view" setting, as the next setting, "Escape key navigates to home view", is related to it. Please refer to the attached screenshots.

<details>
<summary> Screenshots </summary>

| Before - `thinner` class is applied | After - `thinner` class is removed |
|--------------|-------------|
| ![image](https://github.com/user-attachments/assets/f4c25bcd-35b4-420a-9e4a-681bb927ca74) | ![image](https://github.com/user-attachments/assets/8cff8fe2-288b-4532-a2be-d149d533cf42) |

</details>

@alya, so should I also remove the class from there?
Thank you.

timabbott commented:
Yeah let's just remove the `thinner` class.

saubhagya-patel commented:
Following the review received, I have updated this PR. The following changes were made:

- Following [this](https://github.com/zulip/zulip/pull/34389#issuecomment-2810352707) comment, the PR now aims to remove the instances of the `.thinner` CSS class mainly used in settings.
  - We used the `.thinner` class in the following Settings > Preferences (Please refer to the attached screenshots):
  
      - Navigation: ([`web/templates/preferences_navigation.hbs`](https://github.com/zulip/zulip/blob/main/web/templates/settings/preferences_navigation.hbs#L7)) - "Home view"
    - Information: ([`web/templates/preferences_information.hbs`](https://github.com/zulip/zulip/blob/main/web/templates/settings/preferences_information.hbs#L52)) - "Play animated images".

- The [`web/templates/settings/settings_numeric_input.hbs`](https://github.com/zulip/zulip/blob/main/web/templates/settings/settings_numeric_input.hbs#L2) file contained an instance of the `.setting-next-is-related` class. It has been removed as it is no longer needed, since it was previously used in combination with the `.thinner` class.


<details>
<summary> Screenshots </summary>

- In Personal settings > Preferences:

| Description | Before | After |
|-------------|--------|-------|
| "Home view" setting | ![image](https://github.com/user-attachments/assets/f32ae22e-4eab-4901-80e3-276131411606) | ![image](https://github.com/user-attachments/assets/451f8e87-f8ff-43e0-87df-ec191e991057) |
| "Play animated images" setting | ![image](https://github.com/user-attachments/assets/a3ad53e4-7150-4aaf-9d7c-a84d7ef8e36e) | ![image](https://github.com/user-attachments/assets/db78b13d-e32a-4642-b371-61289fca58a0) |

- In Organization settings > Default user settings:

| Description | Before | After |
|-------------|--------|-------|
| "Home view" setting | ![image](https://github.com/user-attachments/assets/a88a1d99-b3ba-4ac0-ad24-fddd14900735) | ![image](https://github.com/user-attachments/assets/6be21010-bec8-479f-96c5-1b280cf4b584) |
| "Play animated images" setting | ![image](https://github.com/user-attachments/assets/5594441e-eb94-49dd-8ac8-915f02150330) | ![image](https://github.com/user-attachments/assets/86724f98-2973-4f47-b067-98e41e79421e) |


</details>

@alya, I have made the required changes. I hope the updated changes look good to you. Please let me know if any further adjustments are required.
Thank you.

alya commented:
Yes, this all seems better. @sahil839 any feedback from your side?

sahil839 commented:
Looks good to me. Would be good to add some context about removing "setting-next-is-related" class and mentioning that it was not useful.

saubhagya-patel commented:
Following the review received, I have updated this PR. The following changes were made:

- Following [this](https://github.com/zulip/zulip/pull/34389#issuecomment-2818011156) comment, to add more context about removing `.setting-next-is-related` class, the commit message is modified to include it.

  - Updated [commit](https://github.com/zulip/zulip/pull/34389/commits/49da6165146c70b93accac185d2d33f6df5733d7) message:
   ```
    css: Remove `.thinner` class from Settings.
    
    This commit removes the `.thinner` class from the
    "Play animated images" setting in Preferences > Information,
    and from the "Home view" setting in Preferences > Navigation.
    The `.setting-next-is-related` class, used in combination with
    the `.thinner` class, is also removed as it is not needed.
    We no longer use the `.thinner` class.
   ```

@sahil839, I have made the required changes. I hope the updated changes look good to you. Please let me know if any further adjustments are required.
Thank you.

sahil839 commented:
Looks good.

timabbott commented:
Merged, thanks @saubhagya-patel and @sahil839!
